<launch>

    <!-- map_provider -->
    <node name="map_provider_node" pkg="map_provider" type="map_provider_node.py" output="screen">
    </node>

    <!-- Gmapping -->
    <node pkg="gmapping" type="slam_gmapping" name="slam_gmapping"/>

</launch>